### ISSUE TITLE
Travis: MacOSX LuaJIT 2.1 job failing

### ISSUE THREAD

**Issue by squeek502** [luvit/luv#446]
Latest git LuaJIT 2.1 branch now [has this commit](https://github.com/LuaJIT/LuaJIT/commit/8961a92dd1607108760694af3486b4434602f8be):

> OSX: Don't set a default MACOSX_DEPLOYMENT_TARGET. 

which is causing the CI to fail: https://travis-ci.org/squeek502/luv/builds/629232852

```
==== Building LuaJIT 2.1.0-beta3 ====
/Applications/Xcode-9.4.1.app/Contents/Developer/usr/bin/make -C src
Makefile:322: *** missing: export MACOSX_DEPLOYMENT_TARGET=XX.YY.  Stop.
make: *** [default] Error 2 
```

**Comment by zhaozg** [comment]
I notice that long time ago, so I make https://github.com/zhaozg/luajit-cmake.
If need, I can make a PR use https://github.com/zhaozg/luajit-cmake replace current cmake build for luajit.

**Comment by squeek502** [comment]
Closed by #447 (see also https://github.com/LuaJIT/LuaJIT/issues/538)
